static/app/components/events/interfaces/spans/scrollbarManager.tsx

```diff
@@ -92,42 +93,6 @@ export class Provider extends Component<Props, State> {
     // but only for DOM elements that actually got rendered
 
     this.initializeScrollState();
-
-    const anchoredSpanHash = window.location.hash.split('#')[1];
-
-    // If the user is opening the span tree with an anchor link provided, we need to continuously reconnect the observers.
-    // This is because we need to wait for the window to scroll to the anchored span first, or there will be inconsistencies in
-    // the spans that are actually considered in the view. The IntersectionObserver API cannot keep up with the speed
-    // at which the window scrolls to the anchored span, and will be unable to register the spans that went out of the view.
-    // We stop reconnecting the observers once we've confirmed that the anchored span is in the view (or after a timeout).
-
-    if (anchoredSpanHash) {
-      // We cannot assume the root is in view to start off, if there is an anchored span
-      this.spansInView.isRootSpanInView = false;
-      const anchoredSpanId = window.location.hash.replace(spanTargetHash(''), '');
-
-      // Continuously check to see if the anchored span is in the view
-      this.anchorCheckInterval = setInterval(() => {
-        this.spanBars.forEach(spanBar => spanBar.connectObservers());
-
-        if (this.spansInView.has(anchoredSpanId)) {
-          clearInterval(this.anchorCheckInterval!);
-          this.anchorCheckInterval = null;
-        }
-      }, 50);
-
-      // If the anchored span is never found in the view (malformed ID), cancel the interval
-      setTimeout(() => {
-        if (this.anchorCheckInterval) {
-          clearInterval(this.anchorCheckInterval);
-          this.anchorCheckInterval = null;
-        }
-      }, 1000);
-
-      return;
-    }
-
-    this.spanBars.forEach(spanBar => spanBar.connectObservers());
```

Comment:
Deleting this block of code, since it is no longer necessary. The new changes allow anchor links to work out of the box without this hack